.github/CODEOWNERS

```diff
@@ -15,10 +15,7 @@
 # Blocks
 /packages/block-library                         @ajitbohra @fabiankaegy
 /packages/block-library/src/gallery
```

Comment:
Should we also delete this line? There are no owners for the Gallery.

Reply:
There are a lot of lines with no code owners. That could be a separate cleanup.